optaplanner-persistence/optaplanner-persistence-jpa/src/test/java/org/optaplanner/persistence/jpa/util/PersistenceUtil.java

```diff
@@ -0,0 +1,324 @@
+/*
+ * Copyright 2011 Red Hat Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.optaplanner.persistence.jpa.util;
+
+import bitronix.tm.BitronixTransactionManager;
+import bitronix.tm.TransactionManagerServices;
+import bitronix.tm.resource.jdbc.PoolingDataSource;
+import org.drools.core.base.MapGlobalResolver;
+import org.drools.core.impl.EnvironmentFactory;
+import org.h2.tools.DeleteDbFiles;
+import org.h2.tools.Server;
+import org.junit.Assert;
+import org.kie.api.runtime.Environment;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Persistence;
+import javax.transaction.UserTransaction;
+import java.io.IOException;
+import java.io.InputStream;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+import static org.junit.Assert.assertNotNull;
+
+public class PersistenceUtil {
```

Comment:
If you have some additional time it would be great to generalize this DB Utils into `kie-test-util` (https://github.com/droolsjbpm/drools/tree/master/kie-test-util) module, which was created exactly for this purpose (having general test-related utils that can be shared across different KIE repositories).